### ISSUE TITLE
Relay 1 connection question

### ISSUE THREAD

**Issue by azhiltsov** [Tristan79/Vento#1]
I am wondering why did you connect Relay 1 in the "N" circuit, while for safety it should be on L3 for "Vento L3" or on L1 for "Vento L1".  Was there any particular reason for that?

**Comment by Tristan79** [comment]
Yeah, you are right. It would be much safer to cut all connections, if you want to power it off completely. No reason, other than I did not think it through, and just cut N instead of all.

**Comment by azhiltsov** [comment]
I was asking mainly because I faced issue with relay got stuck in "ON" position at "MAX" speed and I thought that might be you've done this on purpose. Probably when it switches to the max it discharges capacitor in extraction fan. I might try to change the logic to not keep L1/L3 always connected, but switch it off instead, when changing to higher power.
